pom.xml

```diff
@@ -354,6 +354,9 @@ THE SOFTWARE.
           <groupId>org.apache.maven.plugins</groupId>
           <artifactId>maven-javadoc-plugin</artifactId>
           <version>2.10.3</version>
+          <configuration>
+            <quiet>true</quiet>
```

Comment:
Related only to the extent that I was frustrated when reading the build log for a previous PR head that was marked as a failure due to a Javadoc error to have to scroll through hundreds of lines of junk before I got to the actual warnings & errors.